packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/definecallback.md

```diff
@@ -1,13 +1,15 @@
-Define the following placeholder inside your app's `build.gradle`:
+Parse and define the following placeholder inside your app's `build.gradle`:
```

Comment:
I'm not sure that this new text adds anything. The developer isn't "Parsing" anything, they are adding content to the file that I assume is a placeholder. If that's the case, then the original text is fine. The only thing that may be worth adding is what the placeholder does.

Reply:
The code being added is parsing the manifest placeholder. I'm updating this section to make this more clear. Will be in the next commit.